pkgs/top-level/all-packages.nix

```diff
@@ -9761,6 +9761,10 @@ in
 
   leaps = callPackage ../development/tools/leaps { };
 
+  ### DEVELOPMENT / JAVA MODULES
+
+  javaPackages = recurseIntoAttrs (callPackage ./java-packages.nix { });
```

Comment:
This is going to open up all kinds of weird little things. I won't get around to doing this in the near future... I had hoped to get this into the next release, but I will try to fix this first.